lib/connectors/gitlab/connector.rb

```diff
@@ -22,10 +22,9 @@ class Connector < Connectors::Base::Connector
       def initialize
         super()
         @extractor = Connectors::GitLab::Extractor.new(
-          :base_url => configurable_fields[:base_url][:value],
-          :api_token => configurable_fields[:api_token][:value]
+          :base_url => configurable_fields['base_url'][:value],
```

Comment:
why not symbols?

Reply:
That was my lousy attempt to fix the broken configurable fields validation, I've attempted to do the fix in https://github.com/elastic/connectors/pull/166 instead and will revert this change